### PR TITLE
super-slicer.stable: 2.3.56.8 -> 2.3.56.9

### PR DESCRIPTION
**Pull request by cab404**
also, super-slicer.staging: init at 2.3.57.0

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by cab404** [comment]
Result of `nixpkgs-review pr 136962` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>super-slicer</li>
    <li>super-slicer-staging</li>
  </ul>
</details>

File: pkgs/applications/misc/prusa-slicer/super-slicer.nix [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
       owner = "supermerill";
       repo = "SuperSlicer";
-      sha256 = "sha256-em0OgrcPaV2VYM8DpvtVJjgdojStMF/ROUEtZ8iLZfo=";
       rev = version;
       fetchSubmodules = true;
     };
 
     # We don't need PS overrides anymore, and gcode-viewer is embedded in the binary.
     postInstall = null;
-    dontStrip = true;
```
Comment (Moredread) [inline pkgs/applications/misc/prusa-slicer/super-slicer.nix:20]: Is there a reason to remove seperate debug symbols again?

Comment (cab404) [reply pkgs/applications/misc/prusa-slicer/super-slicer.nix:20]: that's a good question. I thought I've never actually added it in the last PR)

**Comment by Moredread** [comment]
@ofborg build super-slicer super-slicer-staging

**Review by Moredread — CHANGES_REQUESTED** [review]
It seems that ```dontStrip = true;``` is superfluous when using ```separateDebugInfo = true;```, e.g. from the usage in pkgs/development/libraries/glibc/default.nix. I'm not sure if this even skips stripping the binaries, e.g. still increasing the file size. I think it would be best to remove ```dontStrip```

Sorry, I didn't catch this in the previous review.

I've successfully sliced a model with both versions, so besides the issue above it looks good. :)

**Comment by cab404** [comment]
I think I've tried building without "dontStrip", and got unusable symbols in the end. I will try again nonetheless, I may be mistaken -- a lot of flag juggling was taking place)

**Comment by cab404** [comment]
@Moredread yup, it doesn't do anything. Removed it.

**Review by Moredread — APPROVED** [review]

**Review by Moredread — COMMENTED** [review]
I've been thinking about having a seperate 'staging' package. The problem I see with this is that there don't seem to be no real release cycle, i.e. no dedicated staging/preview process. Early versions for a minor version (i.e. 2.3.57.0) seem to be more like an alpha or release candidate in other projects. I.e. after a release is considered stable, releases after that are stable as well for a time.

This would lead to "staging" to potentially be an older versions than "stable". I think I'd be happier with naming it "*-latest" and keeping it always at the latest release.

I'm not sure how other maintainers decide if a separate package for cutting edge releases is appropriate though.

**Comment by cab404** [comment]
> I've been thinking about having a seperate 'staging' package. The problem I see with this is that there don't seem to be no real release cycle, i.e. no dedicated staging/preview process. Early versions for a minor version (i.e. 2.3.57.0) seem to be more like an alpha or release candidate in other projects. I.e. after a release is considered stable, releases after that are stable as well for a time.
> 
> This would lead to "staging" to potentially be an older versions than "stable". I think I'd be happier with naming it "*-latest" and keeping it always at the latest release.
> 
> I'm not sure how other maintainers decide if a separate package for cutting edge releases is appropriate though.

renaming it to latest and occasionally having latest == stable sounds okay. is that what you have in mind?

**Comment by Moredread** [comment]
@cab404 yes :)

**Comment by cab404** [comment]
i guess that would be something we do in the next release then D:

**Pull request merged**